src/Apps/Auction/Components/RegisterButton.tsx

```diff
@@ -33,7 +33,7 @@ export const RegisterButton: React.FC<RegisterButtonProps> = ({ me, sale }) => {
       openAuthModal(ModalType.login, {
         redirectTo: `${saleURL}/register`,
         intent: Intent.registerToBid,
-        copy: "Log in to bid on artworks",
+        copy: "Sign up to bid on artworks",
```

Comment:
Similarly here. We should either show the sign-up form here or revert the copy change.